plugins/static/README.md

```
@@ -0,0 +1,65 @@
+
+`@eggjs/static` is a plugin that has been built-in for egg. It is enabled by default.
+
+## Configuration
+
+`@eggjs/static` support all configurations in [@eggjs/koa-static-cache](https://github.com/eggjs/egg/tree/next/packages/koa-static-cache).
+And with default configurations below:
+
+- prefix: `'/public/'`
+- dir: `path.join(appInfo.baseDir, 'app/public')`
+- dynamic: `true`
+- preload: `false`
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 1

(MD007, ul-indent)